sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/configuration-reference.md

```diff
@@ -36,14 +36,16 @@ Configuration Reference:
 
 #### Schema Inference Config
 
-Note when running queries unless if are interested to get back the raw json payload 
-we recommend setting `spark.cosmos.read.inferSchemaEnabled` to be `true`. 
+When doing read operations, users can specify a custom schema or allow the connector to infer it. When no custom schema is provided and schema inference is not enabled, the connector will return raw json.
+Enabling the schema inference is done by setting `spark.cosmos.read.inferSchemaEnabled` to be `true`.
 
 | Config Property Name      | Default | Description |
 | :---        |    :----   |         :--- | 
-| `spark.cosmos.read.inferSchemaEnabled`     | `false`    | Whether schema inference is enabled or should return raw json. We recommend to enable this option for query  |
-| `spark.cosmos.read.inferSchemaQuery`      | `SELECT * FROM r`    | When schema inference is enabled, used as custom query to infer it for example if you store multiple entities with different schemas within a container and you want to ensure inference only looks at certain document types|
-| `spark.cosmos.read.inferSchemaSamplingSize`      | `1000`    | Sampling size to use when inferring schema |
+| `spark.cosmos.read.inferSchemaEnabled`     | `false`    | Whether schema inference is enabled or should return raw json. We recommend to enable this option for query. |
```

Comment:
the `spark.cosmos.read.inferSchemaEnabled` default is `true` now. please change we missed earlier to update this.

perhaps you can move the description on raw json here.